google/resource_container_cluster.go

```diff
@@ -374,7 +385,19 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 		for i := 0; i < nodePoolsCount; i++ {
 			prefix := fmt.Sprintf("node_pool.%d", i)
 
-			nodeCount := d.Get(prefix + ".node_count").(int)
+			nodeCount := 0
+			if initialNodeCount, ok := d.GetOk(prefix + ".initial_node_count"); ok {
+				nodeCount = initialNodeCount.(int)
+			}
+			if nc, ok := d.GetOk(prefix + ".node_count"); ok {
+				if nodeCount != 0 {
```

Comment:
should we use ConflictsWith instead in the schema? This way, we bring the validation at the plan stage instead of apply